extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -6,4 +6,22 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 
 		return true;
 	}
+
+	/**
+	 * @param WikiPage $page
+	 * @param Revision $revision
+	 * @param $parentRevisionId
+	 * @param User $user
+	 * @return bool
+	 */
+	public static function onNewRevisionFromEditComplete( $page, $revision, $parentRevisionId, $user ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$page, $user". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)